docs/source/migration.mdx

```diff
@@ -391,6 +395,14 @@ class MoviesAPI extends RESTDataSource { //highlight-line
   async getMovie(id: string): Promise<Movie> {
     return this.get<Movie>(`movies/${encodeURIComponent(id)}`);
   }
+
+  async updateMovie(movie: Movie): Promise<Movie>  {
+    return this.patch(
+      `movies`,
```

Comment:
It's correct as is, but typically only use template strings if we need interpolation as well via `${}`
```suggestion
      'movies',
```